catalog/aws-lb-optr-1-2/catalog.yaml

```diff
@@ -0,0 +1,88 @@
+---
+defaultChannel: stable-v1
+name: aws-load-balancer-operator
+schema: olm.package
+---
+entries:
+- name: aws-load-balancer-operator.v1.2.0
+name: stable-v1.2
+package: aws-load-balancer-operator
+schema: olm.channel
```

Comment:
I forgot one thing. We have to define the upgrade graph in the channels. Which CSV the new bundle replaces and which ranges should be skipped. The latter is a quite a serious question, we used to skip all the versions which precede the new one to fast forward all the users to the latest version available. We should discuss which strategy we want to follow and whether the same one makes sense for both [`stable-vX` and stable-vX.Y` channels](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/versioning.md#channels).